lang/system/pl.xml

```diff
@@ -308,44 +556,268 @@ System Vita3K jest gotowy!</completed_setup>
 				<mm_dd_yyyy>MM/DD/RRRR</mm_dd_yyyy>
 			</date_format>
 			<time_format name="Format godziny">
-				<clock_12_hour>Zegar 12-godz</clock_12_hour>
-				<clock_24_hour>Zegar 24-godz</clock_24_hour>
+				<clock_12_hour>Zegar 12-godz.</clock_12_hour>
+				<clock_24_hour>Zegar 24-godz.</clock_24_hour>
```

Comment:
Revert both here, is totaly fine, no using . In end

Reply:
That's Sony's mistake but whatever already, not like I'm grammarly correct irl either.